JortPob/ESM/Papyrus.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json.Nodes;
+using System.Threading.Tasks;
+
+namespace JortPob.ESM
+{
+    public class Papyrus
+    {
+        public readonly string id;
+
+        public Papyrus(JsonNode json)
+        {
+            id = json["id"].GetValue<string>();
+
+            string raw = json["text"].GetValue<string>();
+            string[] lines = raw.Split("\r\n");
+            for(int i=0;i<lines.Length;i++)
+            {
+                string sanitize = lines[i];
+
+                // Remove trailing comments
+                if (sanitize.Contains(";"))
+                {
+                    sanitize = sanitize.Split(";")[0].Trim();
+                }
+
+                // Remove any multi spaces
+                while (sanitize.Contains("  "))
+                {
+                    sanitize = sanitize.Replace("  ", " ");
+                }
+            }
```

Comment:
I noticed that this loop doesn't actually _do_ anything since we don't store `lines` or `sanitize` anywhere outside of the variables here, but this loop could be simplified in one of two ways:

1. You could get rid of it entirely with this regexp:
```c#
// Removes all comments and replaces any group of multiple spaces with a single space
string sanitized = Regex.Replace(line, @"(\s{2,}|;.*)", " ", RegexOptions.Multiline));
```

2. You can remove the inner `while` loop with this loop:
```c#
// Replace any occurrence of multiple spaces with a single space
sanitize = Regex.Replace(sanitize, @"(\s{2,})", " ");
```